.github/workflows/merge-dev-to-main.yml

```diff
@@ -0,0 +1,25 @@
+name: Merge Dev to Main
+
+on:
+  workflow_dispatch:
+
+jobs:
+  merge:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout main branch
+      uses: actions/checkout@v2
+      with:
+        ref: main
+
+    - name: Checkout dev branch
```

Comment:
why have two checkout. don't you just need the one dev checkout?

Reply:
yep, that was from the prev approach, now we only need to checkout to main